tools/test_storage_hyperv_primitive.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+
+import json
+import sys
+
+from cloudcix_primitives import storage_hyperv
+
+cmd = sys.argv[1]
+
+host = None
+domain_path = 'D:\HyperV\primitive_test\\'
```

Comment:
```suggestion
domain_path = 'D:\HyperV\primitive_test\'
```

typo?

Reply:
It is needed otherwise it crashes with:

  File "test_storage_hyperv_primitive.py", line 11
    domain_path = "D:\HyperV\primitive_test\"
                                            ^
SyntaxError: EOL while scanning string literal